src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -506,7 +506,7 @@ namespace SurfaceGeometry {
 
 		}
 
-		CalculateZoneVolume( ErrorsFound, ZoneCeilingHeightEntered ); // Calculate Zone Volumes
+		CalculateZoneVolume( ZoneCeilingHeightEntered ); // Calculate Zone Volumes
```

Comment:
One approach we'd like to move toward is returning the status as the function return value.  Nothing to do for this one, but just thinking about the future, have `calculateZoneVolume` return a boolean where zero is success and non-zero is failure.